Jenkinsfile

```diff
@@ -167,4 +158,44 @@ pipeline {
 	        }
 	    }
 	}
+}
+
+def adaptDarc(String capellaProduct) {
+	//Install Cybersecurity on Capella
+	sh "chmod 755 ./capella/capella"
+	sh "chmod 755 ./capella/jre/bin/java"
+	
+	sh "cp -rp $capellaProduct capella-darc"
+	sh "./capella/capella -application org.eclipse.equinox.p2.director -profile DefaultProfile -destination capella-darc -repository file:/${WORKSPACE}/releng/org.polarsys.capella.cybersecurity.site/target/repository -installIU org.polarsys.capella.cybersecurity.feature.feature.group -noSplash"            
```

Comment:
the -profile DefaultProfile  is ok here, 
indeed, in the generated config.ini, this is the default value used
![image](https://user-images.githubusercontent.com/34767327/105329787-d54fc780-5bd1-11eb-98d4-b925c3433a96.png)